src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
 // write your code here
+const table = document.querySelector('table');
+
+/// создание head
+const createTh = document.createElement('th');
+
+createTh.textContent = table.querySelectorAll('thead tr th')[1].textContent;
+
+const vst = table.querySelectorAll('thead tr th')[3];
+
+vst.after(createTh);
+
+// создание  body
+
+table.querySelectorAll('tbody tr').forEach((elem) => {
+  const createTd = document.createElement('td');
+
+  createTd.textContent = elem.cells[1].textContent;
```

Comment:
Ensure that each row in the body has at least two cells. Accessing `elem.cells[1]` will throw an error if there are fewer than two cells.